hypervisor.go

```diff
@@ -201,6 +203,25 @@ func serializeParams(params []Param, delim string) []string {
 	return parameters
 }
 
+// DeserializeParams converts []string to []Param
+func DeserializeParams(parameters []string) []Param {
+	var params []Param
+
+	for _, param := range parameters {
+		if param == "" {
+			continue
+		}
+		p := strings.SplitN(param, "=", 2)
+		if len(p) == 2 {
+			params = append(params, Param{Key: p[0], Value: p[1]})
+		} else if len(p) == 1 {
+			params = append(params, Param{Key: p[0], Value: ""})
+		}
```

Comment:
Thanks for review.

After strings.SplitN(param, "=", 2), len(p) is either 1 or 2. So another else statement seems like unnecessary. Correct me if I'm wrong.